espnet/nets/beam_search_transducer.py

```diff
@@ -11,7 +11,7 @@
 from espnet.nets.pytorch_backend.transducer.utils import is_prefix
 from espnet.nets.pytorch_backend.transducer.utils import recombine_hyps
 from espnet.nets.pytorch_backend.transducer.utils import select_lm_state
-from espnet.nets.pytorch_backend.transducer.utils import substract
+from espnet.nets.pytorch_backend.transducer.utils import subtract
```

Comment:
This change is the function name, and it would be safe to fix it on the other PR by checking the usage of the function across the module.

Reply:
Thanks for the concern Shinji! I'm okay with the changes and I was going to resolve the conflicts with my PR.
I'll modify it on my side then